src/js/control-bar/volume-control/volume-control.js

```diff
@@ -36,6 +36,23 @@ class VolumeControl extends Component {
         this.removeClass('vjs-hidden');
       }
     });
+
+    this.on(this.volumeBar, ['focus', 'slideractive'], () => {
```

Comment:
this is the change from the volume-menu-button to make changing volume more easily?